src/components/title/styles.ts

```diff
@@ -1,12 +1,12 @@
 import { cva } from "class-variance-authority";
 
-const titleStyles = cva("", {
+const titleStyles = cva("font-medium", {
```

Comment:
Portebbe essere comodo definire una `defaultVariant`, in modo che se non viene passata la prop `as` il componente abbia comunque uno stile di default (es. `h1`)